client/ayon_core/hosts/nuke/api/plugin.py

```diff
@@ -584,14 +587,22 @@ def get_representation_data(
                                     Defaults to False.
             custom_tags (list[str], optional): user inputted custom tags.
                                                Defaults to None.
+            colorspace (str, optional): colorspace name.
+                                        Defaults to None.
         """
         add_tags = tags or []
```

Comment:
Rather add the tag here instead of appending it in the dict.
```suggestion
        add_tags = tags or []
        add_tags.append(self.name.replace("_", "-"))
```